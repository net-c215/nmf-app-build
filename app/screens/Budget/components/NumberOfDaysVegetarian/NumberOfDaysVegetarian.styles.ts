import { StyleSheet } from "react-native";

const styles = StyleSheet.create({
  container: {
    marginTop: 10,
    marginBottom: 26,
    flex: 1,
    alignItems: "center"
  },
  daysContainer: {
    flex: 1,
    flexDirection: "row"
  }
});

export default styles;
