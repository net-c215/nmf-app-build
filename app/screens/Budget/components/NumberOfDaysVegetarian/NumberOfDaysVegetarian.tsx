import React from "react";
import { View } from "react-native";
import { Text } from "../../../../components";
import styles from "./NumberOfDaysVegetarian.styles";

const NumberOfDaysVegetarian = () => {
  return (
    <View style={styles.container}>
      <View style={styles.daysContainer}>
        <Text.H3 green>45 days</Text.H3>
        <Text.H3> without meat!</Text.H3>
      </View>
      <Text.H3>Good job ðŸŽ‰</Text.H3>
    </View>
  );
};

export default NumberOfDaysVegetarian;
