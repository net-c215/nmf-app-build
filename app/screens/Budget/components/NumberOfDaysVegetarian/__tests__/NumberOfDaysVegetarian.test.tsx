import React from "react";
import renderer from "react-test-renderer";
import NumberOfDaysVegetarian from "../NumberOfDaysVegetarian";

jest.unmock("../NumberOfDaysVegetarian");

it("NumberOfDaysVegetarian renders correctly", () => {
  const tree = renderer.create(<NumberOfDaysVegetarian />).toJSON();
  expect(tree).toMatchSnapshot();
});
