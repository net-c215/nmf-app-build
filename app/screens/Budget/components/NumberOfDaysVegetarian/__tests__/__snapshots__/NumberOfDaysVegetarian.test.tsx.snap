// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`NumberOfDaysVegetarian renders correctly 1`] = `
<View
  style={
    Object {
      "alignItems": "center",
      "flex": 1,
      "marginBottom": 26,
      "marginTop": 10,
    }
  }
>
  <View
    style={
      Object {
        "flex": 1,
        "flexDirection": "row",
      }
    }
  >
    <Text.H3
      green={true}
    >
      45 days
    </Text.H3>
    <Text.H3>
       without meat!
    </Text.H3>
  </View>
  <Text.H3>
    Good job ðŸŽ‰
  </Text.H3>
</View>
`;
